process_request/helpers.py

```diff
@@ -3,6 +3,102 @@
 from rapidfuzz import fuzz
 
 
+def get_collection_creator(resource):
+    """Takes json for an archival object that has the _resolved parameter on resource::linked_agents. Iterates through linked_agents; if the role is creator, appends to list, and returns list as a string."""
+    creators = []
+    if resource.get("linked_agents"):
+        for linked_agent in resource.get("linked_agents"):
+            if linked_agent.get("role") == "creator":
+                creators.append(linked_agent.get("_resolved").get('display_name').get('sort_name'))
+    return ",".join(creators)
+
+
+def get_location(top_container_info):
+    """Gets a human-readable location string for a top container
+
+    Args:
+        top_container_info (dict): json for a top container (with resolved container locations)
+
+     Returns:
+         string: all locations associated with the top container, separated by a comma.
+     """
+    locations = []
+    for c in top_container_info.get("container_locations"):
+        locations.append(c.get("_resolved").get("title"))
+    return ",".join(locations)
+
+
+def get_container_field(top_container_info, container_field):
```

Comment:
Is this method really necessary? All you're doing is calling `get()` on a dict...